Makefile

```diff
@@ -7,19 +7,23 @@ help:
 	@echo "clean				Remove artifacts and standardize repo."
 
 # CORE ]------------------------------------------------------------------------
-test:
-	black --check .
+test: deps
```

Comment:
I'm getting an `Error 1` when I run this for the first time.

Reply:
This mirror the exit code of black. I saw that on my branch one of the files is not formatted correctly. I will update it accordingly and then you shouldn't see it anymore.

Reply:
If you want, we can ignore single non-zero codes by adding `-` in front of the command, in this case line 12